packages/manager/core/generator/app/templates/src/components/Loading/Loading.tsx

```diff
@@ -2,5 +2,11 @@ import React from 'react';
 import { OsdsSpinner } from '@ovhcloud/ods-components/spinner/react';
 
 export default function Loading() {
-  return <OsdsSpinner />;
+  return (
+    <div className="flex justify-center">
+      <div>
```

Comment:
On a besoin de cet div en plus ?

Reply:
The Loading components is useless, we can update it or delete it of course